Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM gcr.io/distroless/static:nonroot
+FROM golang:1.19-buster AS build
 
-COPY dependency-track-exporter /
+WORKDIR /app
 
-USER nonroot
-ENTRYPOINT ["/dependency-track-exporter"]
+COPY go.mod ./
+
+COPY go.sum ./
+
+RUN go mod download
+
+COPY *.go ./
+
+COPY internal ./internal
+
+RUN go build -o /dependency-track-exporter
+
+FROM gcr.io/distroless/base-debian11:latest-amd64
```

Comment:
Could we use `gcr.io/distroless/static:nonroot` here? Unless there's a reason we need debian, I'd prefer to use the most minimal possible image.